geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/ExplainCommand.scala

```diff
@@ -24,6 +24,11 @@ class ExplainCommand(parent: JCommander) extends CommandWithCatalog(parent) with
   override def execute() =
     try {
       val q = new Query(params.featureName, ECQL.toFilter(params.cqlFilter))
+      Option(params.attributes).foreach { attribute =>
+        val splitAttrs = attribute.toString().split("""(?<!\\),""").map(_.trim.replace("\\,", ","))
```

Comment:
don't we use this line somewhere else? might be good to pull it out into a common trait